File: Source/Herc.Pwa.Js/source/Edge/Dtos/SendDto.ts
export const enum FeeOption {
  Low = "low",
  Standard = "standard",
  High = "high"
}

export interface SendDto {
  edgeCurrencyWalletId: string;
  destinationAddress: string;
  nativeAmount: string;
  currencyCode: string;
  fee: FeeOption;
}

File: Source/Herc.Pwa.Client/Features/Edge/DTOs/SendDto.cs
namespace Herc.Pwa.Client.Features.Edge.Dtos
{
  public class SendDto
  {
    public string EdgeCurrencyWalletId { get; set; }
    public string DestinationAddress { get; set; }

    public string NativeAmount { get; set; }

    public string CurrencyCode { get; set; }

    public string Fee { get; set; }
  }
}
